lib/str.ts

```diff
@@ -971,15 +971,16 @@ const Str = {
      *  Formats a URL by converting the domain name to lowercase and adding the missing 'https://' protocol.
      *
      * @param url The URL to be formatted
+     * @param scheme The Scheme to use in the URL
      * @returns The formatted URL
      */
-    sanitizeURL(url: string): string {
+    sanitizeURL(url: string, scheme = 'https'): string {
```

Comment:
```suggestion
    sanitizeURL(url: string, defaultScheme = 'https'): string {
```

Reply:
I think `defaultScheme` is more meaningful because it uses only if the URL didn't have a scheme. we also need to update the comment and its usage.

Reply:
We can also add a unit to confirm that
```
expect(Str.sanitizeURL('https://example.com', 'http')).toBe('https://example.com');
```